packages/moonstone/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The following is a curated list of changes in the Enact moonstone module, newest
 ### Fixed
 
 - `moonstone/VideoPlayer` to round the time displayed down to nearest second
+- `moonstone/Picker` to correctly update its value once when button is selected
```

Comment:
This might be too vague to document the change. Perhaps we should mention the change in hold threshold since it could impact apps or testing.